src/02-common-principles.md

```diff
@@ -73,6 +76,12 @@ misunderstanding we clarify them here.
     acquisition parameters and task (however events can change from run to run
     due to different subject response or randomized nature of the stimuli). Run
     is a synonym of a data acquisition.
+    Note that "uninterrupted" may look different by modality due to the nature of the
+    recording.
+    For example, in [MRI](04-modality-specific-files/01-magnetic-resonance-imaging-data.md),
```

Comment:
```suggestion
    For example, in [MRI](04-modality-specific-files/01-magnetic-resonance-imaging-data.md), or [MEG] (04-modality-specific-files/02-magnetoencephalography.md) 
```
This is also the case for MEG? @robertoostenveld 
and EEG? Let's set this clear for all the modalities

Reply:
This section wasn't really intended to be exhaustive so much as to make the contextual nature of this entity clear. Happy to change to MEG if we want to be less MR-centric.

Reply:
Yes, not a priority. Only to have some examples in other modalities too